src/main/java/io/meeds/qa/ui/pages/page/factory/space/SpaceHomePage.java

```diff
@@ -1215,4 +1214,40 @@ public void viewAllRepliesInCommentsDrawer(String comment) {
     getCommentsDrawerViewAllReplies(comment).clickOnElement();
   }
 
+  private BaseElementFacade getReactionActivityList(String activity) {
+    return findByXPathOrCSS(String
+            .format("(//*[contains(text(),'%s')]//following::p[contains(@class,'likersNumber')])[1]", activity));
+  }
+
+  public void clickReactionList(String activity) {
+    getReactionActivityList(activity).clickOnElement();
+  }
+
+  @FindBy(xpath = "//*[contains(@class,'v-list-item__content drawerTitle') and contains (text(),'Reactions')]")
+  private BaseElementFacade reactionDrawer;
+
+  public void drawerDisplay() {
+    reactionDrawer.shouldBeVisible();
+  }
+
+  private BaseElementFacade getUserInDrawerListLikers(String user) {
+    return findByXPathOrCSS(
+            String.format("//aside[contains(@class,'v-navigation-drawer--open')]//p[contains(text(),'%s')]", user));
+  }
+
+  public void hoverAndGoToUSerProfile(String user) {
+    getUserInDrawerListLikers(user).hover(
+            String.format("//aside[contains(@class,'v-navigation-drawer--open')]//p[contains(text(),'%s')]", user));
+    getUserInDrawerListLikers(user).clickOnElement();
```

Comment:
Please avoid calling twice the same method and define a variable for it : 
```suggestion
    BaseElementFacade element = getUserInDrawerListLikers(user);
    element.hover();
    // TODO should add a wait for popover to be opened and a check that it's opened if necessary, else, no need to make a hover here, than a click on it. You may delete completely the hover event if it's not useful for test.
    element.clickOnElement();
```

Reply:
hover event deleted and access to profile event is kept